src/index.html

```diff
@@ -22,6 +22,452 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+      <header class="header">
+        <a
+          class="header__logo"
+          href="/"
+        >
+          <img
+            src="images/logo.png"
+            alt="Logotip"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#section1"
```

Comment:
fix href value everywhere 
```suggestion
                href="#apple"
```